### ISSUE TITLE
repo website is slow to load

### ISSUE THREAD

**Issue by bugzpodder** [r-wasm/webr#306]
https://repo.r-wasm.org/

**Comment by coatless** [comment]
There is an R shinylive app loading in the background for the repo stats. So, there is a bit to download... :'(

**Comment by georgestagg** [comment]
Yes, it's a combination of the time to load the required R packages and parse the repository's `PACKAGES` file.

We're working on improving package loading times, which should help a lot. Especially once packages are cached on a user's machine.
